anda/desktops/ctwm/ctwm.spec

```diff
@@ -0,0 +1,54 @@
+Summary: Lightweight window manager with virtual workspaces
+Name: ctwm
+Version: 4.1.0
+Release: 1
+URL: https://ctwm.org
+BuildRequires: libx11-devel libjpg-devel libxt-devel libxext-devel libxmu-devel libxpm-devel cmake
+Source0: https://www.ctwm.org/dist/%{name}-%{version}.tar.gz
+Source1: %{name}.desktop
```

Comment:
Doesn't exist in the same directory as the spec

Reply:
you probably forgot to copy this from RPMSphere

Reply:
cannot find it in rpmsphere sources ww